README.md

```diff
@@ -48,11 +48,23 @@ Run your project (Cmd+R)
 
 ### Installation (Android)
 
-* Add Gradle configuration changes
+* In `android/setting.gradle`
 
-Run `react-native link react-native-device-info` in your project root.
```

Comment:
That's odd, `react-native link` should properly link to MainApplication.java, that's actually the PR I wrote to react native.